### ISSUE TITLE
[FR] Keep Multiple Versions of Backups

### ISSUE THREAD

**Issue by shotblocker21** [jensstein/oandbackup#81]
It would be really handy to be able to optionally keep multiple versions of backups for each app.  Especially the apks.
Have run into issues after installing one or more updates from a market where I will find something critical is broken or the newer versions are unusable.  Being able to individually restore an app 3 or more versions previous would be very handy in such cases.

**Comment by jensstein** [comment]
yes, this is definitely a planned feature (i think it may have been mentioned before somewhere).
if everything works out well i might be able to make it for the release after the next one.

**Comment by shotblocker21** [comment]
Awesome!
I looked through the issues but didn't see it. Hope this isn't a duplicate.

**Comment by seonwoolee** [comment]
Is this still planned?
